target/linux/mvebu/Makefile

```diff
@@ -12,7 +12,7 @@ FEATURES:=fpu usb pci pcie gpio nand squashfs ramdisk
 SUBTARGETS:=cortexa9 cortexa53 cortexa72
 MAINTAINER:=Imre Kaloz <kaloz@openwrt.org>
 
-KERNEL_PATCHVER:=4.14
+KERNEL_PATCHVER:=4.19
```

Comment:
@sduponch I think, that it's too early for that.

Reply:
If I remember correctly, kernel 4.19 may only become default after openwrt-19.01 has been created (which should be 4.14 for all targets).
So for now, I suppose this patch can be removed.

Reply:
Sorry, doing...